Orange/widgets/data/owfile.py

```diff
@@ -410,6 +410,10 @@ def apply_domain_edit(self):
         else:
             table = self.data
 
+        if not hasattr(table, "domain") or hasattr(table, "domain") and \
```

Comment:
Could you simplify to `table is not None and ...`? As per above branching, `table` can be either a Table or None.